apptainer/builders/cloud/README.md

```diff
@@ -172,7 +172,7 @@ export REPOSITORY_URL=oras://#LOCATION#/#PROJECT_NAME#/#REPOSITORY_NAME#
 Apptainer needs to authenticate to your repository before it can push or pull images. Use this command to authenticate
 
 ```bash
-apptainer remote login \
+apptainer repository login \
```

Comment:
Is it `registry login` per the message in the PR comment, or `repository login`?

Reply:
it is `registry login` , sorry for that